v3/src/components/axis/models/multi-scale.ts

```diff
@@ -121,4 +122,29 @@
     return {data: NaN}
   }
 
+  formatValueForScale(value: number) {
```

Comment:
How about the following comment for this function?
/** To display values for a numeric axis we use just the number of significant figures required to distinguish the value for one screen pixel from the value for the adjacent screen pixel. **/